src/rsz/src/Resizer.i

```diff
@@ -36,6 +36,7 @@
 %{
 
 #include <cstdint>
+#include <fstream>
```

Comment:
why is this needed?

Reply:
When I removed the ptree headers this code no longer compiles.  It was getting fstream indirectly from dpl includes.